certified-connectors/PDFcross/readme.md

```diff
@@ -0,0 +1,21 @@
+
+## PDFcross
+PDFcross is a service for manipulating PDF files, such as merging PDF files into one file and support in Japanese.
+
```

Comment:
@nabe-hiromichi Please refer the [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md) provided for readme.md file, and add required sections.

Reply:
I confirmed the template and fix readme.